src/components/Card.js

```diff
@@ -193,7 +166,7 @@ export default class Card extends React.Component {
         <img src={imgURL} alt="" />
         <CampaignInfo>
           <Bookmark
-            onClick={this.toggleBookmark}
+            toggleBookmark={this.props.toggleBookmark}
```

Comment:
you could save a few chars by naming this prop simply "toggle", since it's clear that it relates to the Bookmark component.